pkg/operator/validations.go

```diff
@@ -46,10 +47,10 @@ func CompareScrapeTimeoutToScrapeInterval(scrapeTimeout, scrapeInterval string)
 		scrapeInterval = "30s"
 	}
 
-	if si, err = model.ParseDuration(scrapeInterval); err != nil {
+	if si, err = model.ParseDuration(string(scrapeInterval)); err != nil {
```

Comment:
same as for the size type PR, let's add a TODO comment for removing it in a future release.